src/oogeso/plots/plots.py

```diff
@@ -99,7 +99,7 @@ def plot_deviceprofile(
         nrows = nrows + 1
     df2 = res.dfDeviceIsOn.unstack("device")[devs]
     dfPrep = res.dfDeviceIsPrep.unstack("device")[devs]
-    timerange = list(res.dfExportRevenue.index)
+    timerange = list(res.dfDeviceIsOn.index.get_level_values("time"))
```

Comment:
Just a note for the future. Python convention is snake case for instances of a class: df_device_is_on, etc. Makes it easier to read :)

Reply:
No need to change anything for now. Just an advice for the future.

Reply:
OK, noted